tests/framework-tests/preferences/js/PageEnhancerTests.js

```diff
@@ -11,25 +11,24 @@ You may obtain a copy of the ECL 2.0 License and BSD License at
 https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 */
 
-// Declare dependencies
 /* global fluid, jqUnit */
 
 (function ($) {
     "use strict";
 
     fluid.defaults("fluid.tests.pageEnhancerTest", {
-        gradeNames: ["fluid.test.testEnvironment", "autoInit"],
+        gradeNames: ["fluid.test.testEnvironment"],
         events: {
             createPageEnhancer: null
         },
         components: {
             settingsStore: {
-                type: "fluid.tempStore"
+                type: "fluid.prefs.tempStore"
             },
             pageEnhancer: {
                 type: "fluid.pageEnhancer",
                 options: {
-                    creatOnEvent: "createPageEnhancer",
+                    creatOnEvent: "createPageEnhancer", // TODO: HORRIBLE typo here
```

Comment:
The TODO is to determine whether it is indeed unnecessary (which one would have to imagine it is)